### PR TITLE
Default Support Added to Reply To and Signature Attributes

### PR DESCRIPTION
**Pull request by Buddhimah**
https://github.com/wso2/product-is/issues/6146

### CODE REVIEW

File: features/event-publisher/org.wso2.carbon.event.output.adapter.server.feature/src/main/resources/conf_templates/templates/repository/conf/output-event-adapters.xml.j2 [hunk 1]
```diff
@@ -98,6 +98,8 @@
         <property key="mail.smtp.port">{{output_adapter.email.port}}</property>
         <property key="mail.smtp.starttls.enable">{{output_adapter.email.enable_start_tls}}</property>
         <property key="mail.smtp.auth">{{output_adapter.email.enable_authentication}}</property>
+        <property key="mail.smtp.signature">{{output_adapter.email.username}}</property>
```
Comment (hasinidilanka) [inline features/event-publisher/org.wso2.carbon.event.output.adapter.server.feature/src/main/resources/conf_templates/templates/repository/conf/output-event-adapters.xml.j2]: Change key to {{output_adapter.email.signature}}

Comment (Buddhimah) [reply features/event-publisher/org.wso2.carbon.event.output.adapter.server.feature/src/main/resources/conf_templates/templates/repository/conf/output-event-adapters.xml.j2]: as per the default configuration signature should be the username it self

Comment (Buddhimah) [reply features/event-publisher/org.wso2.carbon.event.output.adapter.server.feature/src/main/resources/conf_templates/templates/repository/conf/output-event-adapters.xml.j2]: changed

File: features/event-publisher/org.wso2.carbon.event.output.adapter.server.feature/src/main/resources/conf_templates/templates/repository/conf/output-event-adapters.xml.j2 [hunk 2]
```diff
@@ -98,6 +98,8 @@
         <property key="mail.smtp.port">{{output_adapter.email.port}}</property>
         <property key="mail.smtp.starttls.enable">{{output_adapter.email.enable_start_tls}}</property>
         <property key="mail.smtp.auth">{{output_adapter.email.enable_authentication}}</property>
+        <property key="mail.smtp.signature">{{output_adapter.email.signature}}</property>
+        <property key="mail.smtp.replyTo">{{output_adapter.email.replyTo}}</property>
```
Comment (hasinidilanka) [inline features/event-publisher/org.wso2.carbon.event.output.adapter.server.feature/src/main/resources/conf_templates/templates/repository/conf/output-event-adapters.xml.j2]: Change the name of the key {{output_adapter.email.reply_to}}

File: features/event-publisher/org.wso2.carbon.event.output.adapter.server.feature/src/main/resources/conf_templates/org.wso2.carbon.event.output.adapter.default.json [hunk 3]
```diff
@@ -9,5 +9,7 @@
   "output_adapter.email.hostname": "smtp.gmail.com",
   "output_adapter.email.port": "587",
   "output_adapter.email.enable_start_tls": true,
-  "output_adapter.email.enable_authentication": true
-}
\ No newline at end of file
+  "output_adapter.email.enable_authentication": true,
+  "output_adapter.email.signature": "$ref{output_adapter.email.username}",
+  "output_adapter.email.replyTo": "$ref{output_adapter.email.from_address}"
```
Comment (hasinidilanka) [inline features/event-publisher/org.wso2.carbon.event.output.adapter.server.feature/src/main/resources/conf_templates/org.wso2.carbon.event.output.adapter.default.json]: Change the name of the key {{output_adapter.email.reply_to}}

Comment (Buddhimah) [reply features/event-publisher/org.wso2.carbon.event.output.adapter.server.feature/src/main/resources/conf_templates/org.wso2.carbon.event.output.adapter.default.json]: done

File: features/event-publisher/org.wso2.carbon.event.output.adapter.server.feature/src/main/resources/conf_templates/templates/repository/conf/output-event-adapters.xml.j2 (hunk 2 above)
Comment (Buddhimah) [reply features/event-publisher/org.wso2.carbon.event.output.adapter.server.feature/src/main/resources/conf_templates/templates/repository/conf/output-event-adapters.xml.j2]: done

**Pull request merged**